runtime/lua/vim/treesitter/query.lua

```diff
@@ -195,6 +195,12 @@ function M.set(lang, query_name, text)
   explicit_queries[lang][query_name] = M.parse(lang, text)
 end
 
+--- `false` if query files didn't exist or were empty
+---@type {[string]: {[string]: Query|false}}
```

Comment:
Better change this and the one below to: `table<string,table<string, Query|false>>`

Reply:
done